{"uuid": "adf8562b-12b3-44fe-92dd-ab2c087aaae9", "children": ["5fa1d271-494f-4773-aed8-621c5737ca75"], "befores": [{"name": "read_yaml", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'E:\\\\config\\\\oxurl.yaml'\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"E:\\work_pytest\\conftest.py\", line 12, in read_yaml\n    with open(path +r'\\oxurl.yaml', 'r', encoding='utf-8') as file:\n"}, "start": 1589787175586, "stop": 1589787175586}], "start": 1589787175586, "stop": 1589787175839}